woocommerce-payments.php

```diff
@@ -171,11 +171,26 @@ function wcpay_init_subscriptions_core() {
 			$plugin_slug = "$plugin_name/$plugin_name.php";
 
 			// Check if the specified $plugin_name is in the process of being activated via the Admin > Plugins screen.
-			if ( isset( $_GET['action'], $_GET['plugin'], $_GET['_wpnonce'] ) && wp_verify_nonce( wc_clean( wp_unslash( $_GET['_wpnonce'] ) ), "activate-plugin_{$plugin_slug}" ) ) {
-				$action = sanitize_text_field( wp_unslash( $_GET['action'] ) );
-				$plugin = sanitize_text_field( wp_unslash( $_GET['plugin'] ) );
+			if ( isset( $_REQUEST['action'], $_REQUEST['_wpnonce'] ) && current_user_can( 'activate_plugin', $plugin_slug ) ) {
+				$action              = sanitize_text_field( wp_unslash( $_REQUEST['action'] ) );
+				$plugin_be_activated = '';
```

Comment:
I believe `$plugin_be_activated` was supposed to be `$plugin_being_activated` 😄 
An alternative for shorter variable names could be `$activating_plugin`

Reply:
> I believe $plugin_be_activated was supposed to be $plugin_being_activated 😄

😆 haha yes it was. I've updated the variable to `$activating_plugin` like you suggested. [ad98e60](https://github.com/Automattic/woocommerce-payments/pull/6757/commits/ad98e6090fa2b643a86e355cabae0bac12732269)